examples/aws_driver_example/aws_internal_connection_pool_password_warning_mysql_example.ts

```diff
@@ -72,6 +72,9 @@ try {
 } finally {
   await newClient.end();
 }
+// Clean up resources used by the plugins.
+await PluginManager.releaseResources();
+
 // Closes all pools and removes all cached pool connections.
 await provider.releaseResources();
```

Comment:
These two methods are meant to be called at the end of the application. Curious if we should move both of them to the end, after we are done with newClient2.

Reply:
Same comment for the aws_internal_connection_pooling_mysql_example.